project/assets/main/creatures/primary/diota/hard-times.chat

```diff
@@ -10,8 +10,13 @@ diota, d, buttercup_2
 
 d: -_- Oh hello. Did you need something?
 [butts_up] Butt's up with you
+ (link_if chat_finished chat/meet_the_competition or chat_finished creature/jayton/soliciting_buttercup_customers)
 [you_fixed_your_sign] You fixed your sign?
+ (link_if chat_finished chat/meet_the_competition or chat_finished creature/jayton/soliciting_buttercup_customers)
 [are_you_out_of_business] Are you out of business yet
+ (link_if chat_finished chat/meet_the_competition or chat_finished creature/jayton/soliciting_buttercup_customers)
+[we_havent_met]
+ (link_if not (chat_finished chat/meet_the_competition or chat_finished creature/jayton/soliciting_buttercup_customers))
```

Comment:
This should probably use the new 'start_if' conditions, and Diota should greet you in a friendlier way if you haven't met.

Reply:
Fixed to use new 'start_if' conditions. Diota greets you more gently if you haven't met.